pkg/kubelet/util/sliceutils/sliceutils.go

```diff
@@ -49,3 +49,37 @@ func (a ByImageSize) Less(i, j int) bool {
 }
 func (a ByImageSize) Len() int      { return len(a) }
 func (a ByImageSize) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+
```

Comment:
`PodsByCreationTime` is not used anywhere. Do we want to keep it?

Reply:
Removed it.